README.md

```diff
@@ -56,6 +56,22 @@ make serve
 
 This will start the local Hugo server on port 1313. Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
 
+## Rebuilding the API reference pages
+
+The API reference pages located in `content/en/docs/reference/kubernetes-api/ref` are built from the Swagger specifications, using https://github.com/feloy/kubernetes-resources-reference.
+
+If you want to rebuild these pages, you first have to pull the `kubernetes-resources-reference` submodule:
+
+```
+git submodule update --init --recursive --depth 1
+```
+
+Next, to build the pages, run:
+
+```
+make api-reference
+```
+
```

Comment:
**Note:** These steps are required unless we automate the site's build commands in Netlify. 
(See my earlier comment about `make api-reference`.)

It's a good idea to explain the build process for updating API reference docs from end to end:

> To rebuild API reference pages:
> 
> 1. Pull and initialize the `kubernetes-resources-reference` submodule:
> 
> ```
> git submodule update --init --recursive --depth 1
> ```
> 
> 2. Next, build the pages.
> 
> ```
> make api-reference
> ```
> 
> You can test the results locally by making and serving the site from a container image:
> 
> ```
> make container-image
> make container-serve
> ```
> 
> ```